hash2curve/src/hash2field/expand_msg/xmd.rs

```diff
@@ -146,12 +147,40 @@ where
     }
 }
 
+/// Error type for [`ExpandMsgXmd`].
+#[derive(Debug, Clone, Copy, PartialEq, Eq)]
+pub enum ExpandMsgXmdError {
+    /// The domain separation tag is empty.
+    DSTError(EmptyDST),
+    /// The length in bytes is too large.
+    ///
+    /// `len_in_bytes` must be at most `255 * HashT::OutputSize`
+    OversizedLen,
+}
+
+impl core::fmt::Display for ExpandMsgXmdError {
+    fn fmt(&self, f: &mut core::fmt::Formatter<'_>) -> core::fmt::Result {
+        match self {
+            Self::DSTError(e) => write!(f, "{e}"),
+            Self::OversizedLen => write!(f, "length in bytes is too large"),
```

Comment:
```suggestion
            Self::OversizedLen => write!(f, "`len_in_bytes` is too large"),
```
Lets keep consistent so users can figure out what they need here.